v2/googlecloud-to-splunk/src/main/java/com/google/cloud/teleport/v2/templates/GCSToSplunk.java

```diff
@@ -210,8 +210,8 @@ static ReadCsv readFromCsv(GCSToSplunkOptions options) {
   static LineToFailsafeJson convertToFailsafeAndMaybeApplyUdf(GCSToSplunkOptions options) {
     return CsvConverters.LineToFailsafeJson.newBuilder()
         .setDelimiter(options.getDelimiter())
-        .setUdfFileSystemPath(options.getJavascriptTextTransformGcsPath())
-        .setUdfFunctionName(options.getJavascriptTextTransformFunctionName())
+        .setJavascriptUdfFileSystemPath(options.getJavascriptTextTransformGcsPath())
+        .setJavascriptUdfFunctionName(options.getJavascriptTextTransformFunctionName())
```

Comment:
Same here - a python udf test

Reply:
We haven't added python udf here in GCS to splunk. I had to update the naming of this class because we changed it's base class. this class will receive a python update in the coming PRs

Reply:
Python UDF tests will be added when python UDF support comes out for this template.